Makefile

```diff
@@ -1489,6 +1495,36 @@ ironic_deploy_cleanup: namespace ## cleans up the service instance, Does not aff
 	oc rsh -t $(DBSERVICE_CONTAINER) mysql -u root --password=${PASSWORD} -e "flush tables; drop database if exists ironic;" || true
 	oc rsh -t $(DBSERVICE_CONTAINER) mysql -u root --password=${PASSWORD} -e "flush tables; drop database if exists ironic_inspector;" || true
 
+##@ REDIS
```

Comment:
the redis targets, expect the "export ALT_CHECKOUT" part, are already definde in the Makefile